composer.json

```diff
@@ -12,20 +11,24 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0"
+        "php": "^5.5 || ^7.0"
```

Comment:
bit late for this - this should only be done on a major version bump, because otherwise, on php 8, composer just loads an older version of the same series in order to satisfy the constraints, which is not desirable